marklogic-data-hub-central/ui/src/components/header/header.tsx

```diff
@@ -103,11 +104,12 @@ const Header:React.FC<Props> = (props) => {
 
   let userMenu = <div className={styles.userMenu}>
     <div className={styles.username}>{localStorage.getItem("dataHubUser")}</div>
-    <div className={styles.logout}>
-      <Button id="logOut" type="primary" size="default"
+    {/* <div className={styles.logout}> */}
```

Comment:
Should remove this comment out code?

Reply:
@federicoRufrancosMarklogic I cleaned the code